lib/block-supports/layout.php

```diff
@@ -69,7 +69,6 @@ function gutenberg_get_layout_style( $selector, $layout ) {
 		$style .= 'display: flex;';
 		$style .= 'gap: var( --wp--style--block-gap, 0.5em );';
 		$style .= 'flex-wrap: wrap;';
-		$style .= 'align-items: center;';
```

Comment:
This is unrelated right?

Reply:
Yes, but it's not needed. This style is handled with block attributes in some blocks now and eventually will be handled by the `layout.flex` controls. For example if we added the support for `Buttons` it would override the style provided by the block.

Reply:
I think we should remove it once we introduce the possibility to tweak otherwise we'll be breaking existing flex blocks as otherwise, their content won't be centered. (which we don't really have but you could try by allowing switching for group)

Reply:
I reverted for now and will revisit when the time comes.